app/src/main/java/com/antosik/benchproject/app/common/binding/chip/ChipBindingAdapters.kt

```diff
@@ -7,11 +7,11 @@ import com.google.android.material.chip.ChipGroup
 
 @BindingAdapter("genresChips")
 fun ChipGroup.addGenresToChips(genres: List<Genres>?) {
-    if (genres != null) {
-        for (i in genres) {
-            val chip = Chip(context)
-            chip.text = i.name
-            addView(chip)
+    genres?.forEach {
```

Comment:
You can also just return from the function when arg is `null`. `if (genres == null) return` this way it will be easier to understand :D

Reply:
I changed it :D